src/client/content_cao.cpp

```diff
@@ -1422,24 +1422,17 @@ void GenericCAO::updateBonePosition()
 		if (!bone)
 			continue;
 
-		//If bone is manually positioned there is no need to perform the bug check
-		bool skip = false;
-		for (auto &it : m_bone_position) {
-			if (it.first == bone->getName()) {
```

Comment:
Why is it that manually positioned bones *do* need the workaround?

Reply:
Empirical evidence suggests it.

Reply:
"Empirical evidence suggests it." I don't understand what you mean.
Well, I tested the same situation with this optimization step - and the glitch was still there.